dist/parser-helper.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
```

Comment:
Is this by accident or by setup that we commit the transpiled code here as well? 😅

Reply:
This is by setup, since we've forked the repository :)